### PR TITLE
Mepts | 1043 List Active Patients ART without TB

### PR DESCRIPTION
**Pull request by eliazmuchave**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/data/converter/DispensationTypeMdcConverter.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package org.openmrs.module.eptsreports.reporting.data.converter;
+
+import org.openmrs.module.reporting.data.converter.DataConverter;
+
+public class DispensationTypeMdcConverter implements DataConverter {
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/data/converter/DispensationTypeMdcConverter.java:5]: @eliazmuchave what is the difference with this class? api/src/main/java/org/openmrs/module/eptsreports/reporting/data/converter/DispensationTypeConverter.java

**Review by psmatsinhe — COMMENTED** [review]
Please have a look

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/data/converter/DispensationTypeMdcConverter.java (hunk 1 above)
Comment (eliazmuchave) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/data/converter/DispensationTypeMdcConverter.java:5]: For same concept the label can be different. For example, for concept 23730 the label must be 'DT' if it's from Dispensation Type field and 'Dispensa Trimestral (DT)' if it's from MDC

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**